ast27/Parser/tokenizer.c

```diff
@@ -16,9 +16,14 @@
 #include "fileobject.h"
 #include "codecs.h"
 #include "abstract.h"
-#include "pydebug.h"
 #endif /* PGEN */
 
+#if PY_MINOR_VERSION >= 10
```

Comment:
https://docs.python.org/dev/whatsnew/3.10.html#id2 says we can just include Python.h instead, which we already do. So let's only include pydebug.h if PY_MINOR_VERSION < 10...
Actually, I just checked Python 3.5's Python.h and it includes both pydebug and pyarena. So I think we can just remove the troublesome includes entirely?

Reply:
That's possible, I didn't think of that because I don't know C. I'll test that out.